pkgs/applications/networking/p2p/retroshare/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, mkDerivation, fetchFromGitHub
+{ lib, stdenv, mkDerivation, fetchFromGitHub, fetchpatch2
```

Comment:
Why is `fetchpatch2` used here?

Reply:
`fetchpatch` is “deprecated”; `fetchpatch2` is meant to be better but is backwards‐incompatible, so you have to opt in.

Ironically we probably need a `fetchpatch3`: https://github.com/NixOS/nixpkgs/issues/266556